src/lint/collect-spelling-diagnostics.ts

```diff
@@ -34,6 +34,19 @@ let matchers = [
     pattern: /[ \t]+\n/gu,
     message: 'Trailing spaces are not allowed',
   },
+  {
+    // The negative lookbehind is to get a better match location
+    pattern: /(?<=\n)\n\n+/gu,
+    message: 'No more than one blank line is allowed',
+  },
+  {
+    pattern: /(?<=<emu-clause.*>\n)\n\s*<h1>/giu,
+    message: "There should not be a blank line between a clause's opening tag and its header",
```

Comment:
what about the last line of a clause, and its closing tag?

Reply:
Sure, added.